packages/windows/src/RPA/Windows/__init__.py

```diff
@@ -466,7 +466,7 @@ def test_do_some_calculations():
     ROBOT_LIBRARY_SCOPE = "GLOBAL"
     ROBOT_LIBRARY_DOC_FORMAT = "REST"
 
-    def __init__(self):
+    def __init__(self, *args, **kwargs):
```

Comment:
Not sure I get where these args/kwargs are used. Can you point it out?

Reply:
So when initializing the windows library you need a way to set the locators path. The way windows library is initialized in the test_windows.py file.

I noticed that the *args, **kwargs idiom was used in the selenium library so wanted to follow that.

Reply:
Gotcha, now I saw the `kwargs.get("locators_path", None)` line, all good! Still, you can place a `locators_path: Optional[str] = None` in a new `LocatorKeywords.__init__` (and keeping the variable args/kwargs when passing from the Windows library to this class instantiation, like you already did). Don't forget to `super().__init__` the initializer from `LocatorKeywords` into `LibraryContext` too.